hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestOzoneConfigurationFields.java

```diff
@@ -55,7 +55,9 @@ private void addPropertiesNotInXml() {
         HddsConfigKeys.HDDS_SECURITY_PROVIDER,
         OMConfigKeys.OZONE_OM_NODES_KEY,
         OzoneConfigKeys.OZONE_ACL_AUTHORIZER_CLASS_NATIVE,
-        OzoneConfigKeys.OZONE_S3_AUTHINFO_MAX_LIFETIME_KEY
+        OzoneConfigKeys.OZONE_S3_AUTHINFO_MAX_LIFETIME_KEY,
+        "ozone.recon.scm.db.dirs"
```

Comment:
I'm curious: why not use the constant `ReconServerConfigKeys.OZONE_RECON_SCM_DB_DIR`?

Reply:
Fixed.